packages/active-state-mixin/active-state-mixin.ts

```diff
@@ -1,29 +1,25 @@
 import { PropertyValues } from 'lit-element';
 import { DisabledStateInterface } from '@vaadin/disabled-state-mixin';
+import { KeyboardMixin } from '@vaadin/keyboard-mixin/keyboard-mixin.js';
```

Comment:
Nit: should the file suffix be omitted? `single-selection-mixin` and `selection-in-view-mixin` at least seem to import modules without the `.js` suffix

Reply:
Submitted a separate issue to fix that for all the mixins:

https://github.com/vaadin/component-mixins/issues/52